pkg/test/common.go

```diff
@@ -632,18 +583,11 @@ func GetRandomImageComponents(layerSize int) (ispec.Image, [][]byte, ispec.Manif
 	return config, layers, manifest, nil
 }
 
-// These are the 3 vulnerabilities found for the returned image by the GetVulnImage function.
-const (
```

Comment:
Maybe move these to image.go (to be together with the creation of the vulnerable images) instead of upload.go.